src/server/middleware/render-index.js

```diff
@@ -86,6 +86,7 @@ export default function renderIndex(html, css, assetMap, store) {
       window.PASSPORT_STRATEGY="${process.env.PASSPORT_STRATEGY || ""}"
       window.CORE_BACKGROUND_COLOR="${process.env.CORE_BACKGROUND_COLOR || ""}"
       window.CAN_GOOGLE_IMPORT=${canGoogleImport}
+      window.SHOW_TAGS=${process.env.SHOW_TAGS || 0}
```

Comment:
Seems like the other boolean vars above (NOT_IN_USA and ALLOW_SEND_ALL) have 0 as their default if the variable isn't set. But I can change it if that doesn't seem right. (also, this environment variable is also just temporary, so that Tags option won't show up in the admin sidebar until we get the texter tag ui in place)

Reply:
Gotcha. Let's set it to `false` then, since we check explicitly for a `false` value in the admin dashboard.